packages/core/src/metadata/di.ts

```diff
@@ -249,6 +249,8 @@ export interface ContentChildDecorator {
    * **Metadata Properties**:
    *
    * * **selector** - The directive type or the name used for querying.
+   * **descendants** - If `true` (default) include all descendants of the element. If `false` then
```

Comment:
```suggestion
   * * **descendants** - If `true` (default) include all descendants of the element. If `false` then
```

Many asterisks here!